### ISSUE TITLE
Tankers run out of gas

### ISSUE THREAD

**Issue by Rabbit994** [Hoggit/TNN2#19]
Tankers commonly run out of gas. Need a system to refuel them or spawn fresh ones when they run out of fuel. This is particular problem with KC-130 due to hornet popularity.

**Comment by LazyBoot** [comment]
Workaround for now, shoot them when they go to land. (Just don't leave parts around on the airfield)

**Comment by Lugghawk** [comment]
I don't think we can set unlimited fuel, but we can respawn them after they land. This will be a change in the HOGGIT framework, but I'll keep the issue open here to track it.

**Comment by Lugghawk** [comment]
Tracked by: https://github.com/Hoggit/H.O.G.G.I.T/issues/5
